sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -452,6 +452,10 @@ def upload_blob(  # pylint: disable=too-many-locals
             Use of customer-provided keys must be done over HTTPS.
             As the encryption key itself is provided in the request,
             a secure connection must be established to transfer the key.
+        :keyword ~azure.storage.blob.ContainerCpkScopeInfo cpk_scope_info:
+            A predefined encryption scope used to encrypts the data on the service-side.
+            The encryption scope is defined on control plane, while the scope could be
```

Comment:
It is unlikely that the terms "control plane" and "data plane" is something that external customers understand.

Reply:
I will do a docs pass once this PR has been merged to master

Reply:
Docs updated :)